src/components/StaticInfo.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+import FullWidthContainer from './FullWidthContainer';
+import LargeText from './LargeText';
+
+import recycleFacts from '../data/recycling-facts.json';
+
+class StaticInfo extends Component {
+  insertRandomFact() {
+    const randomIndex = Math.floor(Math.random() * recycleFacts.length);
+    return recycleFacts[randomIndex];
+  }
+
+  render() {
+    return(
+      <FullWidthContainer flex={this.props.containerFlex}>
+        <LargeText
+          style={styles.largeText}
+          text={this.insertRandomFact()} />
```

Comment:
Does this reset when you navigate away and back to this page? The original idea is that it would only change once a day, but I'm flexible on that.

Reply:
Oh yeah, and we need to do some research on what kind of info would be useful on this page. I'll try to carve some time out for that

Reply:
It does reset when you navigate away and come back for the search page. We can definitely change it later for a daily fact.